doc/guide/getting-started.md

```diff
@@ -47,7 +47,7 @@ Do not use `--enable-poll` if you are on MacOS, as console polling is broken.
 :::
 
 If you intend to build static executables for servers, then you should
-remove `--enable-shared` to configure the system to use static
+add `--enable-shared=no` to configure the system to use static
```

Comment:
I'd rather the document default be --disable-shared, but oh well.

Reply:
ok, we can do that.